File: facturacion-energia/src/main/java/com/co/facturacion/energia/aplicacion/bean/SolicitudData.java
package com.co.facturacion.energia.aplicacion.bean;

import java.io.Serializable;

public class SolicitudData implements Serializable{

	private static final long serialVersionUID = -4346254103429509439L;
	
	private ClienteBean cliente;
	private FacturaBean factura;
	private PagoBean pago;

	public ClienteBean getCliente() {
		return cliente;
	}

	public void setCliente(ClienteBean cliente) {
		this.cliente = cliente;
	}
	
	public FacturaBean getFactura() {
		return factura;
	}

	public void setFactura(FacturaBean factura) {
		this.factura = factura;
	}

	public PagoBean getPago() {
		return pago;
	}

	public void setPago(PagoBean pago) {
		this.pago = pago;
	}
	
}

File: facturacion-energia-frontend/src/app/modelo/solicitudData.ts
import { ClienteBean } from "./clienteBean";
import { FacturaBean } from "./facturaBean";
import { PagoBean } from "./pagoBean";

export class SolicitudData{

    cliente:ClienteBean;
    factura:FacturaBean;
    pago:PagoBean;

    constructor(cliente:ClienteBean, factura:FacturaBean, pago:PagoBean){

        this.cliente = cliente;
        this.factura = factura;
        this.pago = pago;

    }

}